changelogs/changelog.yaml

```diff
@@ -67,3 +64,7 @@ releases:
         name: splunk_data_inputs_network
         namespace: ansible_collections.splunk.es.plugins.modules
     release_date: "2022-09-07"
+  2.2.0:
```

Comment:
We are bumping up 2.2.0 over 2.0.0?  Where is 2.1.0?
Moreover, this should be a bugfix release.

Reply:
Will change this